vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,328 @@
+
+`vcpkg.json`:
+
+:::code language="json" source="../examples/snippets/manifest-mode-cmake/vcpkg.json":::
+
+You only need to specify your direct dependencies in the `"dependencies"` list. When it runs, vcpkg
+resolves and installs any required transitive dependencies.
+
+## 2 - Integrate vcpkg with your build system
+
+In this step we show you how to integrate vcpkg with CMake or MSBuild, so that your project dependencies get automatically installed or restored whenever you build the project. 
+
+If you're using a different build system, skip to the next step: [3 - Install the manifest dependencies].
+
+### [MSBuild](#tab/msbuild)
+
+> [!div class="nextstepaction"]
+> [Learn more about using vcpkg from MSBuild](../users/buildsystems/msbuild-integration.md)
+
+To use [vcpkg in your MSBuild projects](../users/buildsystems/msbuild-integration.md), run the
+following command:
+
+```Console
+vcpkg integrate install
+```
+
+You only need to run the [`vcpkg integrate install`](../commands/integrate.md#vcpkg-integrate-install)
+command the first time you want to enable MSBuild integration. This enables MSBuild integration for all
+your existing and future projects. Use [`vcpkg integrate remove`](../commands/integrate.md#vcpkg-integrate-remove)
+to remove MSBuild system-wide integration.
+
+This integration method automatically adds vcpkg-installed packages to the following project properties:
+`Include Directories`, `Link Directories`, and `Link Libraries`. Additionally, this creates a post-build action
+that ensures that any required DLLs are copied into the build output folder. This works for all solutions and
+projects using Visual Studio 2015 or newer.
+
+### [CMake](#tab/cmake)
+
+> [!div class="nextstepaction"]
+> [Learn more about using vcpkg from CMake](../users/buildsystems/cmake-integration.md)
+
+To use [vcpkg in your CMake projects](../users/buildsystems/cmake-integration.md), you need to set the
+`CMAKE_TOOLCHAIN_FILE` variable to use vcpkg's CMake toolchain file. The vcpkg toolchain is in
+`%VCPKG_ROOT%/scripts/buildsystems/vcpkg.cmake`, where `%VCPKG_ROOT%` is your vcpkg installation path.
+
+To set the toolchain file use any of these methods:
+
+* Set the `CMAKE_TOOLCHAIN_FILE` in your CMakePresets.json file.
+* Pass `-DCMAKE_TOOLCHAIN_FILE=<path/to/vcpkg>/scripts/buildsystems/vcpkg.cmake` as a parameter in your
+CMake configure call.
+* Set the `CMAKE_TOOLCHAIN_FILE` CMake variable before the first call to `project()` in your
+`CMakeLists.txt` file.
+
+---
+
+## 3 - Install dependencies
+
+If you're using CMake or MSBuild and followed the previous step, you can skip ahead to the next step:
+[4 - Build the project].
+
+If you're using a different build system or want to install the dependencies manually, all you need to do
+is run `vcpkg install` in the directory containing your manifest file.
+
+```console
+PS D:\projects\manifest-example> vcpkg install
```

Comment:
I think it would be good to show the manifest used here so that the user understands why we picked cxxopts/fmt/range-v3, such as by adding a `type vcpkg.json` or `cat vcpkg.json` in front of this.